### PR TITLE
docs(fsxaapi): fSXA* | Documentation update for splitted fsxa-api

### PR DESCRIPTION
**Pull request by deangite**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -184,124 +157,52 @@ There is an enum to use these modes.
...
-Optionally, a remoteProject can be passed. If one is passed, the element will be fetched from this project instead of the default one.
-
-In this example the additional parameters ensure that only the fields `identifier` and `displayName` are in the result set:
+This method fetches an element from the configured CaaS. The `FetchElementParams` object defines options to specify your request. Check `buildCaaSUrl` to know which URL will be used.
 
 ```typescript
-fsxaApi.fetchElement('xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx', 'en-EN', {
-  keys: [{ identifier: 1 }, { displayName: 1 }],
-})
+remoteApi.fetchNavigation({ locale, initialPath, fetchOptions, filterContext, }: FetchNavigationParams): Promise<NavigationData | null>;
```
Comment (henczi-espirit) [inline README.md:205]: this looks like a function definition. I think we should stick with working code examples.

File: README.md [hunk 2]
```diff
@@ -413,26 +316,26 @@ These operators can also be found in the [MongoDB Documentation](https://docs.mo
...
+| ------------------------------------------------------------------------ |---------------------------------------------------------------------------------------------------------------------------------------------------| ------------------------------------------ |
+| [FS_CATALOG]<br />[`Catalog`][fs-catalog]`<`[`Catalog$Card`][fs-card]`>` | <nobr>= `CaaSApi_FSCatalog`</nobr><br />= `CaaSApi_Card[]`                                                                                        | `Section[]`                                |
+| [CMS_INPUT_CHECKBOX]<br />[`Set`][fs-set]`<`[`Option`][fs-option]`>`     | <nobr>= `CaaSApi_CMSInputCheckbox`</nobr><br />= `CaaSApi_Option[]`                                                                               | `Option[]`                                 |
+| [CMS_INPUT_COMBOBOX]<br />[`Option`][fs-option]                          | <nobr>= `CaaSApi_CMSInputCombobox`</nobr><br />= `CaaSApi_Option &#124; null`                                                                     | `Option`                                   |
```
Comment (henczi-espirit) [inline README.md:323]: I think these special characters are not rendered correctly at least when I look at the rich diff in this view here.

**Review by henczi-espirit — CHANGES_REQUESTED** [review]
There are some review issues.

File: README.md [hunk 3]
```diff
@@ -48,265 +48,152 @@ The FSXA-API is subject to the Apache-2.0 license.
...
+It proxies the requested resources to a middleware Rest Api, that does not expose secrets.
+If you want to use it in your server, you can use the `FSXARemoteApi`.
+It can be registered as a Rest Api that can be called from a `FSXAProxyApi` instance.
 
 However, to have a fully running application, we recommend using the FSXA-API in your server as well as in your frontend.
 
-In each case you have to specify the content mode, the configuration and optionally the log level.
-
-The config mode can be `preview` or `release`. It depends on which information you want to get.
+The config mode can be `preview` or `release`. Depending on the state of the content you want to fetch.
```
Comment (konstantin-tkachuk) [inline README.md:69]: ...  release, depending on ...

File: README.md [hunk 4]
```diff
@@ -48,265 +48,152 @@ The FSXA-API is subject to the Apache-2.0 license.
...
-  keys: [{ identifier: 1 }, { displayName: 1 }],
+fsxaApi.fetchElement({
+  id: 'xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx',
+  locale: 'de_DE',
 })
 ```
 
+> **_Note_**<br>
+> The 'raw' CaaS data might be helpful when you want to know the key names that you can filter for.
+> It is possible to access those data directly with the authorization header and the CaaS Url.
```
Comment (konstantin-tkachuk) [inline README.md:190]: that data

File: README.md [hunk 5]
```diff
@@ -48,265 +48,152 @@ The FSXA-API is subject to the Apache-2.0 license.
...
+
 ### fetchByFilter
 
 Returns the matching CaaS data entries.
+It maps the given entries to a more frontend-friendly format.
+Which means that unneccessary fields will be omitted and the structure is simpler.
```
Comment (konstantin-tkachuk) [inline README.md:196]: Unnecessary ...

File: README.md [hunk 6]
```diff
@@ -329,36 +216,40 @@ In this example we search for all elements with the `fsType` equals `Example`. W
...
+  pagesize: 50,
+  additionalParams: { keys: { identifier: 0 } },
+  sort: [{ name: 'fsType', order: 'desc' }],
+})
 ```
 
 The default sorting is by the id descending. Multisort is possible and the first sort param is prioritized over subsequent. The sorting is happening on the raw data.
 
+> **_Attention_**<br>
+> The fieldnames and keys that can be passed to the `fetchByFilter` method have to match the key names that are present in the CaaS data.
```
Comment (konstantin-tkachuk) [inline README.md:237]: fieldnames and keys? Are these not just synonyms in this context?
Shouldn't it be simply `The keys that can be passed...`?

File: README.md [hunk 7]
```diff
@@ -329,36 +216,40 @@ In this example we search for all elements with the `fsType` equals `Example`. W
...
+
 ### fetchProjectProperties
 
 Returns the project properties of the given language.
 
-Expects as input parameter the language abbreviation.
+Expects an object parameter with the language abbreviation property
```
Comment (konstantin-tkachuk) [inline README.md:243]: Expects a parameter object with the locale.

File: README.md (hunk 5 above)
Comment (henczi-espirit) [reply README.md:196]: for the frontend developers unneccessary. Which word, would you like to use here?

File: README.md (hunk 4 above)
Comment (henczi-espirit) [reply README.md:190]: will fix it

File: README.md (hunk 6 above)
Comment (henczi-espirit) [reply README.md:237]: in general yes. In this case I wanted distinguish between fieldnames in e.g. filters and the "additionalKeys" -> I will make it more clear.

File: README.md (hunk 7 above)
Comment (henczi-espirit) [reply README.md:243]: I will take that recommendation

**Pull request merged**